service/worker/logger.py

```diff
@@ -11,10 +11,12 @@ class Logger:
     """A logger specific for the tasks of the Worker"""
 
     def __init__(self):
-        logging.basicConfig(level=os.environ.get("BOPTEST_LOGLEVEL", "INFO"))
         self.logger = logging.getLogger('worker')
-        self.formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        self.logger.setLevel(os.environ.get("BOPTEST_LOGLEVEL", "INFO"))
+        fmt = '%(asctime)s UTC\t%(name)-20s%(levelname)s\t%(message)s'
```

Comment:
This is designed to match the log formatting used in testcase.py. It would be wise to consoldiate the logger implementation at some future point.